### PR TITLE
Initial commit of contribution guidelines

### PR DESCRIPTION
**Pull request by 3MUl0R**
This is an initial commit of contribution guidelines.  A template has also been added for pull requests to go along with guideline suggestions.

### CODE REVIEW

**Comment by olympusMonspeacebeuponhim** [comment]
yeah i dont get what any of that means

**Comment by 3MUl0R** [comment]
It's a just help document essentially.

**Review by TheDormouse — APPROVED** [review]
LGTM - The PR template will help quite a bit.

**Comment by AnuAmun** [comment]
#9 guideline needs to be replaced with tutorial

**Comment by TheDormouse** [comment]
> #9 guideline needs to be replaced with tutorial

I think guidelines makes more sense for this, since this is the criteria that would need to be met in order for your PR to be accepted

**Comment by 3MUl0R** [comment]
Resolves #4

**Comment by AnuAmun** [comment]
#9 nope it should be a GUIDE not a RULE - Pirates of the Carrabean - the code it's like guidelines not actual rules ---- RULES are Rules - guidelines are suggestions

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+#### Solve an issue
+
+Scan through our [existing issues](https://github.com/Wookey-Technologies/magna-carta/issues) to find one that interests you. 
+As a general rule, we don’t assign issues to be resolved by anyone specifically. If you find an issue to work on, you are welcome to open a PR with a fix.
+
+### Making Changes
+
+> **Note**  
+> Incase you missed it, you do not need to know how to make a PR to contribute. Opening a Discussion or an Issue will allow the community to help decide on the best action and someone else can do the heavy lifting of creating the PR.
```
Comment (onexdata) [inline CONTRIBUTING.md:46]: Incase-->In case?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+## Getting started
+
+For more information on how we write markdown files, see [the GitHub Markdown reference](https://github.com/github/docs/blob/main/contributing/content-markup-reference.md).
+
+### Issues & Discussions
+
+#### Create a new issue or Discussion
+
+If you spot a problem in the Magna Carta documents you should first search if an [issue](https://github.com/Wookey-Technologies/magna-carta/issues) or relevant [discussion](https://github.com/3MUl0R/magna-carta/discussions) already exists.
```
Comment (onexdata) [inline CONTRIBUTING.md:32]: Comma after "Magna Carta documents"?

**Review by onexdata — CHANGES_REQUESTED** [review]
2 tiny grammar changes.

**Comment by onexdata** [comment]
> #9 nope it should be a GUIDE not a RULE - Pirates of the Carrabean - the code it's like guidelines not actual rules ---- RULES are Rules - guidelines are suggestions

Can you clarify your comment? Are you saying we should change the term "guideline" to "tutorial", or we should make a tutorial?

**Pull request merged**